rdbms/models/profiles.py

```diff
@@ -0,0 +1,37 @@
+from sqlalchemy import Column, Integer, Text, ForeignKey
+from sqlalchemy.exc import IntegrityError
+
+from .database import Base
+from .database import session
+from .users import Users
+
+class Profiles(Base):
+
+    __tablename__='profiles'
+
+    id = Column(Integer, nullable=False, primary_key=True)
+    user_id = Column(Integer, ForeignKey('users.id'))
+    profiles = Column(Text)
```

Comment:
Should be profile instead of profiles. There is one profile per record and users can have multiple records.